.github/workflows/stm32-common.yaml

```diff
@@ -15,7 +15,7 @@ on:
     paths_ignore:
       - 'cmake/Arduino*'
     branches:
-      - '*'
+      - 'edge'
```

Comment:
why alter this? I don't think it blocks us much

Reply:
Fixed, it was left over from a CI test I was doing..